packages/three-instanced-uniforms-mesh/src/InstancedUniformsMesh.js

```diff
@@ -122,6 +122,10 @@ function setAttributeValue (attr, index, value) {
     }
   } else if (size === 4) {
     attr.setXYZW(index, value.x, value.y, value.z, value.w)
+  } else if (value?.toArray) {
+    value.toArray(attr, index * size)
```

Comment:
I'm not quite understanding how this works, shouldn't this be calling toArray with `attr.array` instead of `attr`?

Reply:
You are right, it needs to be attr.array. Done.